web/frontend/src/components/slam.vue

```diff
@@ -117,18 +118,10 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
 };
 
 const executeMoveOnMap = async () => {
-  if (updatedDest === false) {
-    toast.error('please select a destination to move to');
-    return;
-  }
+  moveClick = !moveClick;
+
   // get base resources
   const baseResources = filterResources(props.resources, 'rdk', 'component', 'base');
-  if (baseResources === undefined) {
```

Comment:
Why is this no longer necessary? Was it just removed because the `moveClick = !moveClick` was moved above? If that is the case, I still might suggest logging here.

Reply:
we can't enter this function if `baseResources === undefined` since we do 
```
let moveClick = $computed(() => (
  filterResources(props.resources, 'rdk', 'component', 'base') !== undefined) && ....
  ```

Reply:
so there is no case to log an error for the undefined baseResources